types/ts-nameof/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for ts-nameof 2.0
+// Project: https://github.com/dsherret/ts-nameof#readme
+// Definitions by: David Sherret <https://github.com/dsherret>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line no-unnecessary-generics
```

Comment:
This generic type is necessary because it's a compile time transformation. Ex. `nameof<MyInterface>()` -> `"MyInterface"`.